From f48e41d1ae5fb24e63db0316e8368523acc9a8b0 Mon Sep 17 00:00:00 2001
From: Xiretza <xiretza@xiretza.xyz>
Date: Sat, 1 May 2021 22:01:22 +0200
Subject: [PATCH] build.gradle: replace deprecated "compile" configuration

It was deprecated in gradle 4.10 and removed in gradle 7.0, see
https://stackoverflow.com/a/66910991/11853962.
---
 build.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.gradle b/build.gradle
index 2bfb4fb..477ad1f 100644
--- a/build.gradle
+++ b/build.gradle
@@ -14,7 +14,7 @@ sourceCompatibility = 8
 targetCompatibility = 8
 
 dependencies {
-  compile fileTree(dir: 'jars', include: ['*.jar'])
+  implementation fileTree(dir: 'jars', include: ['*.jar'])
 
 }
 
-- 
2.31.1

